hack/git_client_side_hooks/pre-push

```diff
@@ -39,7 +39,7 @@ do
       RANGE="$REMOTE_SHA..$LOCAL_SHA"
     fi
     
-    NOT_SIGNED_OFF=$(git rev-list --no-merges --author "$AUTHOR" --grep="^Signed-off-by: " --invert-grep "$RANGE")
+    NOT_SIGNED_OFF=$(git rev-list --no-merges --grep="^Signed-off-by: " --invert-grep "$RANGE")
```

Comment:
Why is this updated?

Reply:
This is a different issue I faced. I didn't mean to commit this here, was planning to open a new issue. Will revert this for now.